### ISSUE TITLE
Tutorial how to deploy the react app

### ISSUE THREAD

**Issue by galex** [cfnz/muirwik-starterapp#3]
Hello,

I wonder how I am supposed to package the app to run it independently from IntelliJ (and gradle).
I am mainly trying to dockerize it so I can run in in AWS.

There's a production=false flag, turning it to true and building doesn't really give me a self contained react app like I thought I would get.

How do you deploy your app to production?

**Comment by cfnz** [comment]
Hi, I don't have much in production yet :-)... but so far I have just run it on tomcat and apache...

Basically the production=true does not do a whole lot other than runs kotlin DCE... the next key part is to run the Gradle webpackProd task which runs webpack with the webpack.config.prod.js. This is the thing that makes the build use the react deployment release rather than the development release and brings the bundle size down to about 1MB from around 15 or so... (some would consider this still quite big on a slow mobile network, but I think it is usually OKish)...

Basically then you just take everything in dist and dump it on a web server and it should work. Looking at it again, it might be nice if the bundle.js went into a js sub-folder, the examples I came across at the time did not do that, but I think I will make this change also...

Note to sure on docker or AWS, but I imagine the dist fold is just a normal webpage with associated javascript, so should be OK.

Regards
Colin
